src/theme/ThemeProvider.tsx

```diff
@@ -9,23 +9,60 @@ export const ThemeProvider: FC<ThemeProviderProps> = ({ children }) => {
   const theme = createTheme({
     palette: {
       background: {
-        paper: '#fef8f8'
+        paper: '#fefefe'
       },
       text: {
-        primary: 'cyan'
+        primary: '#000000'
       },
-      divider: '#97948b'
+      divider: '#a1a1a1'
+    },
+    typography: {
+      button: {
+        textTransform: 'none',
+        fontFamily: 'BlinkMacSystemFont'
+      }
+    },
+    components: {
+      MuiLink: {
+        defaultProps: {
+          color: '#000',
+          fontSize: '20px',
+          fontFamily: 'BlinkMacSystemFont'
+        }
+      }
     }
   });
 
+  theme.typography.h3 = {
+    color: 'black',
+    fontFamily: 'BlinkMacSystemFont',
+    padding: '14px',
+    fontWeight: 'normal',
+    fontSize: '40px'
+  };
+
   theme.typography.h5 = {
     color: 'black',
     fontFamily: 'BlinkMacSystemFont',
-    padding: '13px',
+    padding: '14px',
     fontWeight: 'bold',
     fontSize: '23px'
   };
 
+  theme.typography.body1 = {
```

Comment:
Its one of these body styles that is making the table bold, so probably best to not have bold be the default